brave/brave-ios
# Fix #8166: Wallet Tab Bar

## Summary of Changes
- Remove page controller and use a Tab View / tab bar for Wallet

This pull request fixes #8166

## Submitter Checklist:


## Test Plan:
1. Verify Tab bar shows up, switches pages as expected.
2. Verify `...` button still only shows Portfolio settings (graph, nft, balances) on Portfolio tab.
3. Verify Wallet Settings can be opened.
4. Verify pending request button shows up & works as expected (create a transaction or dapp request)


## Screenshots:

Light | Dark
--|--
![Tab Bar - light mode](https://github.com/brave/brave-ios/assets/5314553/21e2a03c-bbb1-4c10-b956-094262cb5bfd) | ![Tab Bar - dark mode](https://github.com/brave/brave-ios/assets/5314553/d656371c-bbba-4cbf-9e84-b97dfff8ec79)
![Tab Bar scrolled - light mode](https://github.com/brave/brave-ios/assets/5314553/89e591dc-3aba-4d8a-95b4-29a161b115a8) | ![Tab Bar scrolled - dark mode png](https://github.com/brave/brave-ios/assets/5314553/2637588a-024f-484a-8c5e-cf5d556f3ff5)

https://github.com/brave/brave-ios/assets/5314553/b0dbff41-bb59-4b5c-ba23-09cf086b5168



## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `bug` / `enhancement`
